### PR TITLE
Update sql-server-to-sql-on-azure-vm-migration-overview.md

### PR DESCRIPTION
**Pull request by suresh-kandoth**
provide link to topic https://learn.microsoft.com/en-us/sql/relational-databases/databases/manage-metadata-when-making-a-database-available-on-another-server

### CODE REVIEW

**Comment by Jak-MS** [comment]
@markjones-msft 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/migration-guides/virtual-machines/sql-server-to-sql-on-azure-vm-migration-overview.md [hunk 1]
```diff
@@ -151,6 +151,8 @@ Depending on the setup in your source SQL Server, there may be additional SQL Se
 - Database Mail
 - Replication
 
+For a complete list of metadata and server objects that you need to move, refer to the detailed information available in [Manage Metadata When Making a Database Available on Another Server](https://learn.microsoft.com/sql/relational-databases/databases/manage-metadata-when-making-a-database-available-on-another-server).
```
Comment (Jak-MS) [inline azure-sql/migration-guides/virtual-machines/sql-server-to-sql-on-azure-vm-migration-overview.md:154]:
```suggestion
For a complete list of metadata and server objects that you need to move, refer to the detailed information available in [Manage Metadata When Making a Database Available on Another Server](../../relational-databases/databases/manage-metadata-when-making-a-database-available-on-another-server.md).
```

File: azure-sql/migration-guides/virtual-machines/sql-server-to-sql-on-azure-vm-migration-overview.md [hunk 2]
```diff
@@ -151,6 +151,8 @@ Depending on the setup in your source SQL Server, there may be additional SQL Se
 - Database Mail
 - Replication
 
+For a complete list of metadata and server objects that you need to move, refer to the detailed information available in [Manage Metadata When Making a Database Available on Another Server](../../relational-databases/databases/manage-metadata-when-making-a-database-available-on-another-server.md).
```
Comment (Jak-MS) [inline azure-sql/migration-guides/virtual-machines/sql-server-to-sql-on-azure-vm-migration-overview.md:154]:
```suggestion
For a complete list of metadata and server objects that you need to move, refer to the detailed information available in [Manage Metadata When Making a Database Available on Another Server](/sql/relational-databases/databases/manage-metadata-when-making-a-database-available-on-another-server).
```

**Comment by suresh-kandoth** [comment]
#sign-off

**Comment by Court72** [comment]
@suresh-kandoth 

PRMerger will accept sign-off from the listed author only. However, if you are a Microsoft employee authorized to update this content, you can send a request to [techdocprs@microsoft.com](mailto:techdocprs@microsoft.com) for this PR to be merged. Include your role and an explanation of why the PR should be merged without approval from the listed author.

**Pull request closed without merging**